packages/dso-toolkit/src/components/legend-item/legend-item.mixins.scss

```diff
@@ -0,0 +1,28 @@
+@use "../../variables/colors";
+@use "../../variables/units";
+@use "legend-item.variables" as legend-item-variables;
+
+@mixin symbolContainer() {
+  background-color: colors.$wit;
+  display: block;
+  float: left;
+  height: legend-item-variables.$symbol-height;
+  margin-inline-end: units.$u1;
+  width: legend-item-variables.$symbol-width;
+  overflow: hidden;
+  position: relative;
+}
+
+@mixin symbol() {
```

Comment:
Kan voor het stylen van het symbol geen samenwerking met het Label worden gemaakt? Label heeft ook ondersteuning voor symbol.

Overigens moeten we ons misschien ook nog even afvragen of het wel aan de toolkit is om dit symbool te stijlen.

Reply:
Hmm, ik pas dezelfde trucendoos toe als het label-component doet door dezelfde styling te hanteren:

```
// HTML/CSS
.dso-label {
  @include label.root();

  .dso-label-symbol {
    @include label.symbolContainer();

    > span {
      @include label.symbol();
    }
  }
}

// Web Component
dso-label {
  span[slot="symbol"] {
    @include label.symbolContainer();

    > span {
      @include label.symbol();
    }
  }
}
`
Om de ruimte voor het symbool te creeeren binnen het Web Component is de tweede selector. Het lukt mij niet om dat binnen de scss van het web-component te realiseren.

Reply:
Check. De toolkit is dus verantwoordelijk voor stijlen van de symbool container.

Deze mixins zijn nu wel dubbel uitgevoerd. Wat vinden we daarvan?

Reply:
Ik heb de mixins gecentraliseerd en deels geparameteriseerd. Label en Legend-item gebruiken ze nu vanaf de centrale plek

Reply:
Keurig!